### ISSUE TITLE
Moderators not getting notified about a flagged post

### ISSUE THREAD

**Issue by KrishaTerranIO** [Hylozoic/hylo-evo#1718]
Hi All, not sure if this is a bug or if flagging is behaving the way it should be. I was testing community moderation for the video I'm creating and found that when a post is flagged, I (being the moderator) did not get a notification about it. I refreshed the page and still nothing came through. Is this a bug? Or is this something that mods don't get notified about?

**Comment by thomasgwatson** [comment]
Yes, unfortunately I either misconfigured the moderation notifications/messages or used an old broken version of them.

I just cleared a post and it stayed in the moderation log... Did something else happen?

We explicitly limited the new moderation actions to 'clear' because of trying to keep the new feature set as small as possible. 

If you can only see delete and not 'remove from group' that is likely expected, because its likely a post you are the author of. 

If you are a mod and its not your post, you should see

![Screenshot 2024-11-08 at 1 47 56 PM](https://github.com/user-attachments/assets/ad754337-e254-4aeb-a2ed-86e3c01dc402)

**Comment by tibetsprague** [comment]
I am working on this On Nov 8, 2024, at 1:49 PM, Tom Watson ***@***.***> wrote:﻿
Yes, unfortunately I either misconfigured the moderation notifications/messages or used an old broken version of them.
I just cleared a post and it stayed in the moderation log... Did something else happen?
We explicitly limited the new moderation actions to 'clear' because of trying to keep the new feature set as small as possible.
If you can only see delete and not 'remove from group' that is likely expected, because its likely a post you are the author of.
If you are a mod and its not your post, you should see
Screenshot.2024-11-08.at.1.47.56.PM.jpg (view on web)

**Comment by KrishaTerranIO** [comment]
I deleted a post I wasn't an author of I didn't see the "Remove this Post" option, only delete. BUT I believe I was the one who flagged the post. I will test this again and update.  The post cleared from the moderation log all together, and I received "there's nothing to see here."

We explicitly limited the new moderation actions to 'clear' -- This makes sense to me.

Ok, another issue, having some strange behavior with information clearing in the mod log then reappearing again. See video attached. 

https://github.com/user-attachments/assets/e2f1865c-c538-4858-a943-a60580d3c7ed

afterwards, if I click on the stream, there is a flagged post in the stream that doesn't show up in the mod log. 

![Screenshot 2024-11-09 at 10 09 41 AM](https://github.com/user-attachments/assets/9ef526f0-c26b-4fc1-8b9a-f1350b475e87)

**Comment by KrishaTerranIO** [comment]
I was able to reproduce the issue. The Mod log only displays when I click the flag on a flagged post. If I just click on the Decisions tab, nothing displays.

Uploading wonkyness with mod log.mov…

**Comment by thomasgwatson** [comment]
The decisions tab includes both proposals and moderation actions, so when
you select decisions you are seeing the absence of proposals. There is a
drop-down that allows you access to the moderation actions.

**Comment by KrishaTerranIO** [comment]
Gotcha, thanks Tom!
